src/app/(app)/missing-shoes/components/MissingShoesHeader.tsx

```diff
@@ -45,9 +45,16 @@ export const MissingShoesHeader = ({ breadcrumbs }: Props) => {
               onChange={handleSearchQueryChange}
               className={`pr-4 py-2 h-7 ${
                 isTitleHide ? "w-32 pl-10" : "w-2 pl-6"
-              } md:w-60 md:pl-10 rounded text-xs`}
+              } md:w-60 md:pl-10 rounded text-xs transition-all duration-500 ease-in-out`}
+            />
+            <MagnifyingGlassIcon
+              onClick={() => setIsTitleHide(true)}
+              className={`absolute top-1/2 ${
+                isTitleHide ? "left-3" : "left-1/2"
```

Comment:
`isTitleHide` seems wrong. @ahmadzulfiqar142 you may guide Nadir how to handle this the correct way.